content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx

```diff
@@ -214,7 +215,7 @@ const Stop: FC<{
                 properties: ['margin-bottom'],
               }}
             >
-              <Divider color="warmNeutral.400" isKeyline={true} />
+              <Divider color="warmNeutral.400" isKeyline={false} />
```

Comment:
Is it worth keeping this if we're not displaying it?

Reply:
agree – just delete the line?